utils/common/simd_sse.h

```diff
@@ -1855,6 +1869,277 @@ class SimdFilterProcessor<
   }
 };
 
+template <SIMD_TYPE simdType>
+vi128_t add(vi128_t x, vi128_t y)
+{
+  switch (simdType)
```

Comment:
I am not certain at all that this switch works the way you expect it. @drrtuy , what do you think?